bids/layout/bids_validator.py

```diff
@@ -252,3 +256,151 @@ def conditional_match(self, expression, path):
                 return False
         else:
             return False
+
+
+def validate_sequences(layout, config):
+    """Checks files in BIDS project match user defined expectations.
+
+    This method is a wrapper for the duplicate_check() and 
+    expected_file_check() methods. Use it to check whether there are
+    files with duplicate content within the BIDS data set and to check
+    the number of data set files against a user customized configuration
+    file. Returns a named tuple of three data frames: duplicates, summary, and problems.
+
+
+    Parameters
+    ----------
+        layout: BIDSLayout class
+            A BIDSLayout path of a data set.
+
+        config: string
+            Path to customized configuration file.
+
+
+    Examples
+    --------
+    >>> layout = bids.grabbids.BIDSLayout('/path_to/sample_project_root')
+    >>> dfs = validate_sequences(layout, 'pybids/bids/layout/tests/data/sample_validation_config.json')
+    >>> dfs.duplicates
+    # Put example output here
+    >>> df.summary
+    # Put example output here
+    >>> df.problems
+    # Put example output here
+    """
+    
+    duplicate_file_df = duplicate_check(layout)
+    summary_df, problem_df = expected_file_check(layout, config)
+    output = namedtuple('output', ['duplicates', 'summary', 'problems'])
+    return output(duplicate_file_df, summary_df, problem_df)
+    
+    
+def duplicate_check(layout):
```

Comment:
For the sake of being able to find all the checking functions easily by name, let's rename these to `check_duplicate_files` and `check_expected_files`.